nuget.config

```diff
@@ -6,7 +6,36 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="VS-Platform" value="https://pkgs.dev.azure.com/devdiv/DevDiv/_packaging/VS-Platform/nuget/v3/index.json" />
+    <!-- CPS components including: Microsoft.VisualStudio.ProjectSystem -->
+    <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
+    <add key="vs-impl-archived" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl-archived/nuget/v3/index.json" />
+    <!-- Microsoft.VisualStudio components -->
+    <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
+    <add key="vssdk-archived" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk-archived/nuget/v3/index.json" />
+    <!--
+      Various packages including:
+        Microsoft.DiaSymReader.Pdb2Pdb
+        Microsoft.CodeAnalysis
+        Microsoft.CodeAnalysis.Features
+        NuGet.VisualStudio
+    -->
+    <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
+    <!--
+      Various packages including:
+        xunit
+        xunit.assert
+        xunit.extensibility.core
+        xunit.extensibility.execution
+        XliffTasks
+        Microsoft.VisualStudioEng.MicroBuild.Plugins.SwixBuild
+    -->
+    <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
+    <!--
+      Public components including:
+        Nerdbank.GitVersioning
+        Codecov
+    -->
+    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
```

Comment:
I don't think we can add this. @MiYanni does that seem right?

Reply:
This is copy of what is in main so it should be fine. Testing out that build works with same feeds as main to make sure we are not using anything internal.

(Changed this file for some testing)